### PR TITLE
Add doc for d2l-colors.

### PR DESCRIPTION
**Pull request by dbatiste**
I can include screenshots for the rest of the palette (all the pluses/minuses, for all the colors) if we want.  The color swatch is ported from the original, as a Lit element.

### CODE REVIEW

File: components/colors/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Colors
+
+## d2l-colors
+
+Importing `d2l-colors` will add the color palette to the document and polyfill for IE11 usinf [shadyCSS](https://github.com/webcomponents/shadycss). Run the demo page to see the full palette.
```
Comment (dlockhart) [inline components/colors/README.md]: Technically not called `d2l-colors` anymore.

File: components/colors/demo/colors.html [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+				<d2l-color-swatch name="malachite-plus-1"></d2l-color-swatch>
+				<d2l-color-swatch name="malachite"></d2l-color-swatch>
+				<d2l-color-swatch name="malachite-minus-1"></d2l-color-swatch>
+				<d2l-color-swatch name="malachite-minus-2"></d2l-color-swatch>
+			</d2l-demo-snippet>
+
+		</d2l-demo-page>
+
+	</body>
+</html>
```
Comment (dlockhart) [inline components/colors/demo/colors.html]: Worth doing a perceptual diff of the full palette?

File: components/colors/README.md (hunk 1 above)
Comment (dbatiste) [reply components/colors/README.md]: Good point - will update.

File: components/colors/demo/colors.html (hunk 2 above)
Comment (dbatiste) [reply components/colors/demo/colors.html]: Oh interesting, yeah, it's probably worthwhile.  Easy enough to add.

Comment (dbatiste) [reply components/colors/demo/colors.html]: Added visual-diff for the entire palette (one screenshot).  It's the largest screenshot/diff that we've done so far, taller than the view-port.  I made it one screenshot in order to save a few seconds, and it should be easy enough to inspect the differences.  If need-be, we can break this up.

Comment (dbatiste) [reply components/colors/demo/colors.html]: ... https://s3.ca-central-1.amazonaws.com/visualdiff.gaudi.d2l/screenshots/colors/2019-05-15%2019.52.02.554/report.html

:)

Comment (svanherk) [reply components/colors/demo/colors.html]: Looks awesome!

Comment (dlockhart) [reply components/colors/demo/colors.html]: Nice!!

**Pull request merged**